frontend/kesaseteli/employer/src/__tests__/utils/components/render-page.tsx

```diff
@@ -0,0 +1,40 @@
+import AxiosTestContext from 'kesaseteli/employer/__tests__/utils/backend/axios-test-context';
+import AuthProvider from 'kesaseteli/employer/auth/AuthProvider';
+import BackendAPIContext from 'kesaseteli/employer/backend-api/BackendAPIContext';
+import Footer from 'kesaseteli/employer/components/footer/Footer';
+import Header from 'kesaseteli/employer/components/header/Header';
+import { NextPage } from 'next';
+import { NextRouter } from 'next/router';
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import Content from 'shared/components/content/Content';
+import Layout from 'shared/components/layout/Layout';
+import theme from 'shared/styles/theme';
+import { ThemeProvider } from 'styled-components';
+import { render, RenderResult } from 'test-utils';
+
+const renderPage = (
+  Page: NextPage,
+  client: QueryClient,
+  router: Partial<NextRouter> = {}
+): RenderResult =>
+  render(
+    <BackendAPIContext.Provider value={AxiosTestContext}>
+      <QueryClientProvider client={client}>
+        <ThemeProvider theme={theme}>
+          <Layout>
+            <Header />
+            <Content>
+              <AuthProvider>
+                <Page />
+              </AuthProvider>
+            </Content>
+            <Footer />
+          </Layout>
+        </ThemeProvider>
```

Comment:
`AuthProvider` wraps the component which is rendered only when logged in, otherwise it will redirect to login page. Layout with header and footer should be shown always and should not be wrapped.

Reply:
I meant here moving `AuthProvider` up to wrap the all visual components

Reply:
It cannot wrap stuff that can be seen unauthenticated like Header